src/platform/forms-system/src/js/components/ContactInfo.jsx

```diff
@@ -0,0 +1,397 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+import { Element } from 'react-scroll';
+
+import {
+  focusElement,
+  scrollTo,
+  scrollAndFocus,
+} from '@department-of-veterans-affairs/platform-utilities/ui';
+import environment from '@department-of-veterans-affairs/platform-utilities/environment';
+
+import {
+  selectProfile,
+  isLoggedIn,
+} from '@department-of-veterans-affairs/platform-user/selectors';
+
+// import { generateMockUser } from 'platform/site-wide/user-nav/tests/mocks/user';
+import { generateMockUser } from '../../../../site-wide/user-nav/tests/mocks/user';
+
+// import { AddressView } from '@department-of-veterans-affairs/platform-user/exports';
+// import AddressView from '@@vap-svc/components/AddressField/AddressView';
+import AddressView from '../../../../user/profile/vap-svc/components/AddressField/AddressView';
```

Comment:
Trying to get these `imports` from `@department...` to work, but I'm seeing `Unable to resolve path to module` errors. Leaving these in here as comments since it takes so long to figure out the correct import. I did merge in [PR #24214](https://github.com/department-of-veterans-affairs/vets-website/pull/24214), but it didn't seem to fix this issue.

Reply:
The struggle is real 😆

Reply:
I did discover lots of instances like this... 😿 

```js
// eslint-disable-next-line import/no-unresolved
import { recordEvent } from '@department-of-veterans-affairs/platform-monitoring/exports';
```

which does work, but eslint doesn't think it does.